app/models/game.rb

```diff
@@ -61,13 +61,13 @@ def to_s
   def popular_categories
     categories.joins(:runs).group('categories.id')
               .having('count(runs.id) >= ' + (Run.where(category: categories).count * 0.05).to_s)
-              .order('count(runs.id) desc')
+              .order(Arel.sql('count(runs.id) desc'))
```

Comment:
Is this the new way to do all raw SQL?

Reply:
What do you mean with all raw SQL?  I'm not really sure why Rails has an issue with this statement, but the depreciation notice said that if you can guarantee that there are no user provided values to wrap it in `Arel.sql` to solve to depreciation.